### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by PCMRStudio** [Yannicked/node-cue-sdk#1]
```
.\js\keyboard.js:7
var cue = new CueSDK();
          ^

TypeError: CueSDK is not a function
    at Object.<anonymous> (.\js\keyboard.js:7:11)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3
```

**Comment by Yannicked** [comment]
My first guess would be that you haven't installed/required the library.
Could you provide some context?

**Comment by PCMRStudio** [comment]
It's that you put the example wrong, it should be `CueSDK.CueSDK()`

**Comment by Yannicked** [comment]
Sorry, made a little mistake in the readme

Fixed in: a12e1611176e7e196a39674e7bf546bbf7247282
